MfciPkg/Application/MfciPolicy/MfciPolicy.py

```diff
@@ -0,0 +1,207 @@
+# @file
+#
+# Write MFCI packet into the MFCI mailbox
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+import os
+import sys
+import logging
+import argparse
+import ctypes
+from UefiVariableSupport.UefiVariablesSupportLib import UefiVariable
+
+gMfciVendorGuid = "EBA1A9D2-BF4D-4736-B680-B36AFB4DD65B"
+
+CurrentPolicyBlob = "CurrentMfciPolicyBlob"
+CurrentMfciPolicy = "CurrentMfciPolicy"
+NextPolicyBlob = "NextMfciPolicyBlob"
+
+MfciPolicyInfo = ["CurrentMfciPolicyNonce", "NextMfciPolicyNonce", "Target\\Manufacturer", "Target\\Product", "Target\\SerialNumber", "Target\\OEM_01", "Target\\OEM_02"]

```

Comment:
coding convention for constant variable names are all capital letters and underscores:
https://peps.python.org/pep-0008/#constants